container-core/src/main/java/com/yahoo/jdisc/http/server/jetty/JettyConnectionLogger.java

```diff
@@ -108,6 +113,14 @@ public void onOpened(Connection connection) {
         });
     }
 
+    private void addReferenceToSslEngine(SocketChannelEndPoint endpoint, ConnectionInfo info, SSLEngine sslEngine) {
+        if (sslEngine != null) {
```

Comment:
Can this be?

Reply:
At least for ALPN, but probably not for SSL connection.